README.md

```diff
@@ -51,10 +51,16 @@ url = "https://example.com"
 user_id = "@user:example.com"
 ```
 
-## Installation (via `crates.io`)
+## Installation (building)
 
 Install Rust (1.83.0 or above) and Cargo, and then run:
 
+From the directory containing the sources (ie: from a git clone):
```

Comment:
I'd move this to another section ("Installation (from local checkout)" or something like that). `cargo install` here just builds and copies the binary to a certain install path.